freemarker/src/main/java/org/jdbi/v3/freemarker/package.html

```diff
@@ -0,0 +1,23 @@
+<!--
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<html><body>
+<p>
+<code>freemarker</code> provides advanced string template processing
+that <code>Jdbi</code> can drive with the statement's bound parameters
+and context to drive statement generation that may need to bind
+variable number of parameters or other advanced use cases.
```

Comment:
This sentence is confusing to me. Also, the template engine has to do with defined attributes, not bound parameters. Those are handled solely by the SqlParser.

Reply:
copy paste is like a box of chocolate, you never what you gonna get ;)
ill try to fix it ;)

Reply:
i added a poor mans fix. Not sure if its good enough or what I really should write, but its hopefully better.